appeals/web/src/server/lib/mappers/appeal/submappers/agent.mapper.js

```diff
@@ -0,0 +1,46 @@
+import { permissionNames } from '#environment/permissions.js';
+import { formatServiceUserAsHtmlList } from '#lib/service-user-formatter.js';
+import { mapActionComponent } from '../../component-permissions.mapper.js';
+
+/**
+ * @typedef {import('#appeals/appeal-details/appeal-details.types.js').WebAppeal} WebAppeal
+ * @typedef {import('../../../../app/auth/auth-session.service.js').SessionWithAuth} SessionWithAuth
+ */
+
+/**
+ * @typedef {(
```

Comment:
As mentioned elsewhere, it's a minor point, but the format of these type definitions should ideally follow the established convention